src/pages/nft/fiat/stripe.vue

```diff
@@ -45,7 +45,11 @@ export default {
     },
   },
   asyncData({ error, query }) {
-    const { class_id: classId, payment_id: paymentId } = query;
+    const {
+      class_id: classId,
+      payment_id: paymentId,
+      claiming_token: claimToken,
```

Comment:
Need to change https://github.com/likecoin/likecoin-api-public/blob/master/src/routes/likernft/fiat/stripe.ts#L202 for this to work